barretenberg/cpp/src/barretenberg/smt_verification/circuit/subcircuits.cpp

```diff
@@ -2,23 +2,35 @@
 
 namespace smt_subcircuits {
 
-CircuitSchema get_standard_range_constraint_circuit(size_t n)
+CircuitProps get_standard_range_constraint_circuit(size_t n)
 {
     bb::StandardCircuitBuilder builder = bb::StandardCircuitBuilder();
-    uint32_t a_idx = builder.add_variable(bb::fr::random_element());
+    uint32_t a_idx = builder.add_variable(bb::fr(0));
     builder.set_variable_name(a_idx, "a");
-    builder.create_range_constraint(a_idx, n);
-    return unpack_from_buffer(builder.export_circuit());
+
+    size_t start_gate = builder.get_num_gates();
+    builder.decompose_into_base4_accumulators(a_idx, n);
+    size_t num_gates = builder.get_num_gates() - start_gate;
+
+    CircuitSchema exported = unpack_from_buffer(builder.export_circuit());
+    return { exported, start_gate, num_gates, { 2 }, { num_gates - 1 } };
```

Comment:
Pleas add this 2 as a contant or make a comment inside brackets {/\*like_this\*/ 2}. Same in the standard logic circuit